tests/cypress/stubs/sessionStubs.js

```diff
@@ -16,7 +16,7 @@ export const loginStub = customUser => ({
   cases: {
     success: {
       status: SUCCESS_CODE,
-      response: { user: customUser || user() }
+      response: { user: customUser || userBuilder() }
```

Comment:
This way I think this might be a better way to evaluate if customUser is undefined or null, otherwise it might have some trouble resolving that scenario.

```suggestion
      response: { user: customUser != null ? customUser : userBuilder() }
```